### PR TITLE
Chat history

### PR DESCRIPTION
**Pull request by GuilhermeFaga**
Chat UI made in HTML

- Log players messages and connect/disconnect logs
- 'Enter' key now focus/defocus the chat text input

Captura de Tela 2019-10-02 às 15 30 04

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	<!--Socket.io latest links can be found at https://cdnjs.com/libraries/socket.io-->
 </head>
 
-<body>
+<body oncontextmenu="return false">
```
Comment (valkyrienyanko) [inline public/index.html]: What does this do?

Comment (valkyrienyanko) [inline public/index.html:24]: What does this do?

**Review by valkyrienyanko — CHANGES_REQUESTED** [review]
Can you please run `yarn lint:fix` from the project root directory?

File: public/index.html (hunk 1 above)
Comment (GuilhermeFaga) [reply public/index.html:24]: Cancel context menu from appearing when right clicking the page.

**Pull request merged**